SPECS/openssh/openssh.spec

```diff
@@ -182,8 +175,8 @@ useradd test -G root -m
 sudo -u test -s /bin/bash -c "PATH=$PATH TEST_SSH_UNSAFE_PERMISSIONS=1 make tests"
 
 %pre server
-getent group sshd >/dev/null || groupadd -g 50 sshd
-getent passwd sshd >/dev/null || useradd -c 'sshd PrivSep' -d %{_sharedstatedir}/sshd -g sshd -s /bin/false -u 50 sshd
+getent group sshd >/dev/null || groupadd --system sshd
+getent passwd sshd >/dev/null || useradd --system --comment 'Privilege Separated SSH' --home-dir %{_sharedstatedir}/sshd --gid sshd --shell /bin/false sshd
```

Comment:
@christopherco I vaguely recall a bug being handled by your team regarding predictable group ids for common system groups, but I can't find it right now. Are there any concerns about not having a fixed group id for sshd?

Reply:
I've dug a little bit and I see that Mariner sets `SYS_UID/GID_MIN` to `101` (thereby reserving `1-100` for hard-coded UIDs/GIDs).  I'm personally not a fan of hardcoding UIDs/GIDs since it's all too easy to create clashes by mistake which are hard to spot until someone installs a specific pair of packages and gets cryptic error messages. 

Depending on specific GIDs/UIDs is also a pretty exciting extension of the API of Mariner/the package (you can _never_ change the hard-coded ID without potentially breaking something).

Reply:
I totally agree with your reasoning for the change, and the 2.0 release is definitely the time to make breaking changes like this. I just wanted to loop in the proper people here who can confirm that the change won't be problematic for us.

Reply:
Had a chat with @christopherco and we agree that it would be best to have a non-fixed GID here. This looks good as-is, to me.